alot/db/manager.py

```diff
@@ -385,6 +385,9 @@ def get_threads(self, querystring, sort='newest_first'):
         assert sort in self._sort_orders
         q = self.query(querystring)
         q.set_sort(self._sort_orders[sort])
+        if exclude_tags:
```

Comment:
If you change the default arg to `exclude_tags=()` (not `[]`!!) then you could leave out the ` if` here.